main.py

```diff
@@ -1,10 +1,56 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as players_card_file:
+        data_players = json.load(players_card_file)
+        for data_player in data_players.items():
+            player = data_player[1]
+
+            # race
+            race_name = player["race"]["name"]
+            if Race.objects.filter(name=race_name).exists():
```

Comment:
use here and all of the code get_or_create - https://www.queworx.com/django/django-get_or_create/